### PR TITLE
Release cycle doc

### PR DESCRIPTION
**Pull request by elibud**
Added a new file that contains a brief description of how our release cycle works. 

@astralbodies @diegoreymendez can you take a look? If you find anything you would like to change feel free to edit directly. Thanks.

### CODE REVIEW

File: RELEASE-CYCLE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+### Day 14: SUBMISSION & RELEASE
+
+- Fetch the localized strings from GlotPress and integrate them into the project.
+- Generate a production build and upload it to the store and phase release it.
+- Finalize the release on GitHub and close the milestone.
+- Merge `release/9.1` into `develop` and into `master`.
+
+## Hot Fix
+
+Sometimes there is a bug or crash that can’t wait two weeks to be fixed. This is how we handle this, for example when a critical issue is uncovered on version `9.0` of the app, currently released.
```
Comment (diegoreymendez) [inline RELEASE-CYCLE.md]: For consistency, even if minor, I'd suggest we'd stick to using a single example version number (for example: 9.1 throughout the whole document).

**Comment by diegoreymendez** [comment]
The document looks mostly fine, but I'm not sure I understand it's purpose in it's current form and location (ie: inside our repo).

What's the target audience for this documentation?

If it's aimed at Automatticians, then there are some details and steps that we won't be able to add here because they refer to internal processes.  This could cause our documentation to become fragmented.

If it's aimed at external contributors, then I'd consider trimming it down to the basics of how our process affects contributions.  For instance, I don't think external contributors would find much value in steps like "Fetch the localized strings from GlotPress and integrate them into the project.".

My thought is that maybe having this kind of documentation is fine, but that it doesn't make sense in this repo.

**Comment by elibud** [comment]
@diegoreymendez the purpose of the doc is to inform contributors about how we do things, not aimed at Automatticians at all. We had a brief talk when I wrote the internal post with @catehstn and @nheagy and we figured this was a good place to drop a quick summary of how we do things. 
Re: Strings -> This is one of the more frequent question I get from new contributors so I think it makes sense to keep it here.

File: RELEASE-CYCLE.md (hunk 1 above)
Comment (elibud) [reply RELEASE-CYCLE.md]: Fixed, thanks!

**Pull request merged**